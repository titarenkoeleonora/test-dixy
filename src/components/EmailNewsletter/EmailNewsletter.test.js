import EmailNewsletter from './EmailNewsletter';
import renderer from 'react-test-renderer';

it(`Correctly EmailNewsletter render`, () => {
  const tree = renderer
    .create(
      <EmailNewsletter />
    );

  expect(tree).toMatchSnapshot();
});