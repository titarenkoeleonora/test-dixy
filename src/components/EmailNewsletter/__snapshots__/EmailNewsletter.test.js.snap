// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly EmailNewsletter render 1`] = `
<section
  className="email-newsletter"
>
  <div
    className="container"
  >
    <div
      className="email-newsletter__wrapper"
    >
      <h2
        className="email-newsletter__title"
      >
        Подписаться на рассылку
      </h2>
      <p
        className="email-newsletter__text"
      >
        Значимость этих проблем настолько очевидна, что постоянный количественный рост и сфера нашей активности позволяет оценить значение новых предложений.
      </p>
      <form
        className="newsletter-form email-newsletter__form"
        method="post"
      >
        <input
          className="newsletter-form__input-text input-text"
          name="name"
          placeholder="Имя"
          required={true}
          type="text"
        />
        <input
          className="newsletter-form__input-text input-text"
          name="email"
          placeholder="E-mail"
          required={true}
          type="email"
        />
        <button
          aria-label="Подписаться"
          className="newsletter-form__button"
          type="submit"
        >
          Подписаться
        </button>
      </form>
    </div>
  </div>
</section>
`;
